src/SignalR/common/Http.Connections.Common/src/NegotiationResponse.cs

```diff
@@ -5,14 +5,44 @@
 
 namespace Microsoft.AspNetCore.Http.Connections
 {
+    /// <summary>
+    /// A response to a '/negotiate' request.
+    /// </summary>
     public class NegotiationResponse
     {
+        /// <summary>
+        /// An optional Url to redirect the client to another endpoint.
+        /// </summary>
         public string Url { get; set; }
+
+        /// <summary>
+        /// An optional access token to go along with the Url.
+        /// </summary>
         public string AccessToken { get; set; }
+
+        /// <summary>
+        /// The public ID for the connection.
+        /// </summary>
         public string ConnectionId { get; set; }
+
+        /// <summary>
+        /// The private ID for the connection.
+        /// </summary>
         public string ConnectionToken { get; set; }
+
+        /// <summary>
+        /// The minimum value between the version the client sends and the maximum version the server supports.
+        /// </summary>
         public int Version { get; set; }
+
+        /// <summary>
+        /// A list of transports the server supports.
+        /// </summary>
         public IList<AvailableTransport> AvailableTransports { get; set; }
+
+        /// <summary>
+        /// An optional error during the negotiate. If this is not null the other properties on the response can be ignored.
```

Comment:
Did I mess the nullability on these?

Reply:
Yeah, we missed it, oh well

Reply:
We didn't enable nullable here, did we?

Reply:
We have annotation on a lot of projects

Reply:
But not this one, right? Because if we did, I would just mark it nullable.

Reply:
Don't know if that would fly for 5.0 RTM, we can do it in 6.0

Reply:
Oh if it's not annotated, that's cool. We never claimed to be 100% nullable. I imagine this isn't a layer users mess around with, is it?

Reply:
We skipped netstandard packages didn't we?